doc/source/dev/contributor/quickstart_mac.rst

```diff
@@ -80,7 +80,7 @@ Currently we have *two* versions of SciPy: the latest release as installed by An
 
 #. Enter ``conda install cython numpy pytest spyder pybind11``. |br| Note that we're only installing SciPy's build dependencies (and Spyder so we can use the IDE), but not SciPy itself.
 
-#. Enter ``conda develop /scipy``, where ``scipy`` is to be replaced with the full path of the SciPy root directory. |br| This instructs ``conda`` to add the root SciPy directory to the |PYTHONPATH|_ environment variable whenever our ``scipydev`` virtual environment is activated. That way, when we ``import`` SciPy code, the code is imported from our development version of SciPy.  *Note: this step differs from the process shown in* `Anaconda SciPy Dev: Part II (macOS)`_ *, which involves creating special folders and files deep within the Anaconda directory. You can ignore that part of the video from 0:38 to 1:38; this is much simpler!*
+#. Enter ``conda develop /scipy``, where ``scipy`` is to be replaced with the full path of the SciPy root directory. |br| This will allow us to ``import`` the development version of SciPy in Python regardless of Python's working directory. *Note: this step replace the steps shown in* `Anaconda SciPy Dev: Part II (macOS)`_ *that modify the ``PYTHONPATH`` environment variable when the ``scipydev`` virtual environment is activated. You can ignore that part of the video from 0:38 to 1:38; this is much simpler!*
```

Comment:
```suggestion
#. Enter ``conda develop /scipy``, where ``scipy`` is to be replaced with the full path of the SciPy root directory. |br| This will allow us to ``import`` the development version of SciPy in Python regardless of Python's working directory. *Note: this step replaces the steps shown in* `Anaconda SciPy Dev: Part II (macOS)`_ *that modify the ``PYTHONPATH`` environment variable when the ``scipydev`` virtual environment is activated. You can ignore that part of the video from 0:38 to 1:38; this is much simpler!*
```